pandas/_libs/algos.pyx

```diff
@@ -678,7 +678,7 @@ def backfill_2d_inplace(ndarray[algos_t, ndim=2] values,
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def arrmap(ndarray[algos_t] index, object func):
+def arrmap(algos_t[:] index, object func):
```

Comment:
does const not work with fused types?

Reply:
I could not make it compile. Seems there's an open [issue](https://github.com/cython/cython/issues/1772) about it in the cython repo.